docs/deployment/deployment-local.md

```diff
@@ -0,0 +1,73 @@
+---
```

Comment:
As explained in my comment in the getting started file below, I think you should not even bother explaining anything about reverse proxy, SQL, Redis, etc.... In my mind, this bundle is dedicated to the getting started which is here to just deploy an instance of Authelia with the minimum explanation so that the user can get glimpse of what Authelia looks like for real in less than a minute. The user can simply test and know whether this is what fits their needs. It's a pure showcase. And if they are happy they can proceed with local where all details will be provided.